### DISCUSSION TITLE
ItemContainerStyle of ThemeResource  can't work on Android

### DISCUSSION THREAD

**Discussion by DJDQfff** [unoplatform/uno#13617] [General]
I meet am error when using          ` ItemContainerStyle="{ThemeResource GridViewItemRevealStyle}"`
* The error willl meet on Android , Windows is Ok. Other platform haven't been tried.
  
* The ItemContainerStyle Property will not work on Android platform, and GridView Control will show nothing
  
* If remove it ,then the GridViewControl can work.
  
* Only tested this one in GridView, don't know how others are

https://github.com/DJDQfff/ItemContainerStyleError.git

**Comment by jeromelaban** [comment]
Thanks for the report. You're mentioning an error, which is it? Can you provide a screenshot or build log?

**Reply by DJDQfff** [reply]
Simply put, if one of my GridView control sets the ItemContainerStyle property  (  ` ItemContainerStyle="{ThemeResource GridViewItemRevealStyle}" `  ), the GridView won't be able to display content on Android

**Reply by Dvij30** [reply]
Hey, I just got familiar with the UNO platform and just started to contributing in it so I am not much aware about what is exactly needed and what are the exact Requirements but what I understand till moment and looking into this error what I can suggest is As you've mentioned, the ItemContainerStyle you are using might be specific to the Windows platform and may not be compatible with Android. If the ItemContainerStyle is crucial for your application, consider creating platform-specific styles for it. Define different styles for Windows and Android and apply them conditionally based on the platform. If I am wrong I would love to hear some productive feedbacks to improve my knowledge also.

**Reply by jeromelaban** [reply]
@MartinZikmund does this look familiar? Maybe it's caused by `GridViewItemRevealStyle` that is not implemented or using unsupported controls ?
